submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -1,71 +1,119 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product Page</title>
+    <link href="../css/productpage.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
You can use just one link instead of three. Try and figure out how to use multiple font family in one link @Onyinye101